packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -295,9 +296,22 @@ class AndroidStudio implements Comparable<AndroidStudio> {
       }
     }
 
+    // Query Spotlight for unexpected installation locations.
+    final ProcessResult spotlightQueryResult = globals.processManager.runSync(<String>[
```

Comment:
does a process exception here get caught? or are we confident mdfind will be present and work?

Reply:
Added a `try`.